docs/manual/scala/guide/advanced/IntegratingNonLagom.md

```diff
@@ -16,27 +16,45 @@ To use the Lagom service interface, you will need to add a dependency on the Lag
+
+The [`StandaloneLagomClientFactory`](api/com/lightbend/lagom/scaladsl/client/StandaloneLagomClientFactory.html) variant is useful for applications that do not have a running `ActorSystem`. Internally, the `StandaloneLagomClientFactory` will create a dedicated `ActorSystem` and `Materializer` to be used by the clients. Both will be managed by the `StandaloneLagomClientFactory` and will be shutdown when the factory is closed.
+
+For example, here's a `StandaloneLagomClientFactory` built using the static service locator, which uses a static URI:
+
+@[static-service-locator-standalone](code/IntegratingNonLagom.scala)
+
+The [`LagomClientFactory`](api/com/lightbend/lagom/scaladsl/client/LagomClientFactory.html) variant is useful for applications that have a running `ActorSystem`. In which case, we recommend to reuse the existing `ActorSystem` and `Materializer`.
 
-The constructor to `LagomClientApplication` takes two arguments, the first is the name of the client, this name is used for the client to identify itself to these services.  The second argument is a `ClassLoader`.
+The easiest way to reuse your existing [[ActorSystem]] and Akka Stream [[Materializer]] is to extend the [[LagomClientFactory]] and add a constructor where you can pass them as arguments.
 
-When you have finished with the application, for example, when the system shuts down, you need to stop the application, by invoking the `stop()` method:
+For example, here’s a `LagomClientFactory` built using the static service locator, which uses a static URI, and reusing an [[ActorSystem]] and [[Materializer]] created outside it:
+
+@[static-service-locator](code/IntegratingNonLagom.scala)
+
+When you have finished with the application, for example, when the system shuts down, you need to stop the factory, by invoking the `stop()` method:
 
 @[stop-application](code/IntegratingNonLagom.scala)
 
-Typically this application will be a singleton in your system.  If your system is using Spring for example, you would create a `FactoryBean` that instantiates it, and you would implement a `@PreDestroy` annotated method that stopped the client application.
+This is dependent on the variant you choose and if your implementation is managing any resources. For instance, the `StandaloneLagomClientFactory` is responsible for managing an internal `ActorSystem` and Akka Streams `Materializer` and therefore its `stop()` method will make sure that they are shutdown. The `LagomClientFactory` on the other hand is not managing any resource and its `stop()` method has a noops implementation.
```

Comment:
I'd rather we didn't say `its `stop()` method has a noops implementation` in the docs since that may not be true in the future. At same time it gives the impression that users may decide to not call it because it's noop after all.

Hmm, is it really noop? There's netty resources, right?